### ISSUE TITLE
Building / deb package for ARM32 platform 

### ISSUE THREAD

**Issue by hongquan** [timescale/timescaledb#1009]
Hi. I want to use TimescaleDB on ARM32 (armhf), but your repo doesn't have deb package. I want to help build. Could you please give the build script (`debian` folder)?

I don't want to use Docker image, to save disk space.

**Comment by Ngalstyan4** [comment]
Hi @hongquan,

You can build and install TimescaleDB on your ARM32 by following the build and install instructions in the readme [here](https://github.com/timescale/timescaledb#user-content-building-from-source-unix-based-systems). I have tested this on a RaspberryPi.

**Comment by hongquan** [comment]
Hi @Ngalstyan4 
That README tells to build from source. But with that method, I have to build on **every** Pi machine and on every Pi machine, I have to install **extra libs**.

What I mean is to use 1 Pi to build and package to _*.deb_ file. After that, I just copy deb file to other Pi and install. No need to wait for building from source (save time) and no need to install extra lib (save disk space).

**Comment by HVP1621** [comment]
Hi, 
I am facing a similar issue, although I intend to build a `.deb` package for `arm64`. I have cloned the timescaledb repo and trying to create a `.deb` package using the `dh_make` and `dpkg-buildpackage` tools, however, I am not able to successfully build the same. For more context, I will list down the steps I followed to create a .deb file:

1. Clone the Github repo, checkout to branch 1.7.2
2. Rename the directory to timescaledb-1.7.2 and create a timescaledb-1.7.2.tar.gz out of the same
3. Pass the above tar file to `dh_make` which gives me a warning: 
`Currently, there is not top level Makefile. This mayrequire additional tuning`
4. I see that there is a `bootstrap` script in the repo which sets up the build env, but I am not sure how to reorganize the scripts so that I have a top-level Makefile
5. Edit the `./debian/control` file to include the dependencies - which I assume should be just `postgresql-12`
6. I anyways attempt to proceed with `dpkg-buildpackage` and the building of the `deb` file fails subsequentlly.

Any pointers towards what all am I getting wrong or how I should be doing things differently would be very helpful!!
